apps/api/src/dotenvcreate.mjs

```diff
@@ -0,0 +1,105 @@
+import { SecretsManagerClient, GetSecretValueCommand } from '@aws-sdk/client-secrets-manager';
+import { existsSync, readFileSync, writeFileSync } from 'fs';
+import { resolve, dirname } from 'path';
+import { fileURLToPath } from 'url';
+
+console.time('dotenvcreate');
+
+const secretName = process.argv[2];
+const region = process.argv[3];
+const enterprise = process.argv[4];
+const env = process.argv[5];
+
+if (!enterprise || enterprise.toLowerCase() === 'false') {
+  console.log('Enterprise value is false or not provided. Exiting script.');
+  process.exit(0);
+}
+
+const secretsManagerClient = new SecretsManagerClient({
+  region,
+});
+// Get the directory of the current script
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+// Function to retrieve secret value
+async function getSecretValue(secretName) {
+  try {
+    const command = new GetSecretValueCommand({ SecretId: secretName });
+    const data = await secretsManagerClient.send(command);
+
+    // Check if the secret value is a string or binary
+    if (data.SecretString) {
+      return JSON.parse(data.SecretString);
+    } else {
+      // Handle binary secret value
+      const buff = Buffer.from(data.SecretBinary, 'base64');
+
+      return JSON.parse(buff.toString('ascii'));
+    }
+  } catch (err) {
+    console.error('Error retrieving secret:', err);
+    throw err;
+  }
+}
+
+// Function to escape or quote values for .env format
+function escapeValue(value) {
+  // If the value contains special characters or spaces, quote it
```

Comment:
You can also just do `value || ''.trim()` to remove leading and trailing spaces.

Reply:
ok

Reply:
I wanted to handle more things like backslash and other things this should cover all